espanso/src/cli/cmd.rs

```diff
@@ -49,7 +50,7 @@ fn cmd_main(args: CliModuleArgs) -> i32 {
     IPCEvent::ToggleRequest
   } else if cli_args.subcommand_matches("search").is_some() {
     IPCEvent::OpenSearchBar
-  } else if cli_args.subcommand_matches("search").is_some() {
+  } else if let Some(subcommand_matches) = cli_args.subcommand_matches("search") {
```

Comment:
Why add an unused variable (and then disable the lint suggesting that you don't do so)? Shouldn't this be left as `is_some()` like the others?

Reply:
Yeah, but `clippy` went mad if I left this as it is. We are checking if `cli_args` is `search` twice, and also we should take the path of `match`, which exhausts every possible option.
This section is more a work in progress, I think